Scarb.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "alexandria"
+version = "0.1.0"
+description = "Starknet utilities"
+homepage = "https://github.com/keep-starknet-strange/alexandria/"
+
+[[target.starknet-contract]]
+
+[dependencies]
+starknet = ">=2.0.0-rc0"
```

Comment:
What about to set `starknet = ">=2.0.0"` or even `starknet = ">=2.0.1"` since release candidates led to build the 2.0.0 final version?